front/src/modules/ui/layout/board/components/BoardColumn.tsx

```diff
@@ -52,6 +43,32 @@ const StyledNumChildren = styled.div`
   width: 16px;
 `;
 
+const StyledButtonGroup = styled.div`
+  align-items: center;
+  display: none;
+  flex-direction: row;
+  margin-left: auto;
+`;
+
+const StyledHeader = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  height: 24px;
+  justify-content: left;
+  margin-bottom: ${({ theme }) => theme.spacing(2)};
+  width: 100%;
+  &:hover {
```

Comment:
we avoid doing this nesting css style. Instead, could you try to use onMouseEnter and onMouseLeave to update a state (useState)